Prettier 2 released, breaks XO

After upgrading Prettier to `v2.0.1` XO started crashing:

```sh
$ xo --fix
Error: Failed to load plugin 'prettier' declared in 'CLIOptions': Cannot find module 'eslint-plugin-prettier'
Require stack:
- /project/__placeholder__.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)
    at Function.resolve (internal/modules/cjs/helpers.js:83:19)
    at Object.resolve (/project/node_modules/eslint/lib/shared/relative-module-resolver.js:44:50)
    at ConfigArrayFactory._loadPlugin (/project/node_modules/eslint/lib/cli-engine/config-array-factory.js:959:39)
    at /project/node_modules/eslint/lib/cli-engine/config-array-factory.js:848:33
    at Array.reduce (<anonymous>)
    at ConfigArrayFactory._loadPlugins (/project/node_modules/eslint/lib/cli-engine/config-array-factory.js:844:22)
    at ConfigArrayFactory._normalizeObjectConfigDataBody (/project/node_modules/eslint/lib/cli-engine/config-array-factory.js:667:32)
    at _normalizeObjectConfigDataBody.next (<anonymous>)
    at ConfigArrayFactory._normalizeObjectConfigData (/project/node_modules/eslint/lib/cli-engine/config-array-factory.js:596:20)
```

--- a/package.json
+++ b/package.json
@@ -86,7 +86,7 @@
 		"open-editor": "^2.0.1",
 		"p-reduce": "^2.1.0",
 		"path-exists": "^4.0.0",
-		"prettier": "^1.15.2",
+		"prettier": "^2.0.4",
 		"resolve-cwd": "^3.0.0",
 		"resolve-from": "^5.0.0",
 		"semver": "^7.1.3",

